go/tools/builders/env.go

```diff
@@ -176,15 +176,15 @@ func runAndLogCommand(cmd *exec.Cmd, verbose bool) error {
 // expandParamsFiles looks for arguments in args of the form
 // "-param=filename". When it finds these arguments it reads the file "filename"
 // and replaces the argument with its content.
-func expandParamsFiles(args []string) ([]string, error) {
+func expandParamsFiles(args []string) ([]string, bool, error) {
```

Comment:
Can you also update the doc of this function for the additional `bool` return value?

Reply:
Done.